lars-reimann/api-editor
# feat: add docstrings in generated python code

Closes #514.

### Summary of Changes

Create docstrings for generate Python classes and functions if they contain any additional information.

### Testing Instructions

1. Launch the backend:
```sh
cd api-editor
./gradlew run
```
2. Open the browser at [localhost:4280](http://localhost:4280).
3. Load some API data into the editor
4. Click "Generate adapters" in the menu bar
5. Extract the downloaded `.zip` file
6. Check the create Python files (`adapters` folder)
